docs/configuration.rst

```diff
@@ -249,6 +249,7 @@ MongoDB
 - ``MONGODB_PASSWORD`` (default: ``""``)
 - ``MONGODB_PORT`` (default: ``27017``)
 - ``MONGODB_USERNAME`` (default: ``""``)
+- ``MONGODB_CONNECT`` (default: ``true``)
```

Comment:
Let's not add a new Tutor configuration setting. Instead, we can default to False for everyone. Can you confirm that setting `connect=False` resolves your issue?